flan/src/routes/(main)/+page.svelte

```diff
@@ -56,14 +67,14 @@
 				<button
```

Comment:
If you make this button a type submit, you don't need a onclick because the default action of that button is to submit the form

Reply:
Addressed